.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-commit": "npm run prettier"
```

Comment:
Please make sure that you add the respective script to `package.json` as well.

Reply:
Thanks for the suggestions! I saw `pretty-quick` before, but totally forgot it was a thing :D I've added it and updated it to run `pretty-quick` instead of `prettier` :) @jamesgeorge007